renderer/canvas.py

```diff
@@ -76,13 +76,17 @@ def _draw_food_image(self, surface, food_name, obs, position):
             position (np.array): (x, y) position of the food (with pix_square_size factor accounted for)
         """
         food_image_name = food_name
-        # Check if cut or cooked
-        for literal in obs:
+        # Check if cut or cooked or fried
+        for literal in obs:        
             if literal.predicate == "iscut" and literal.variables[0] == food_image_name:
                 food_image_name = "cut" + food_image_name
             if literal.predicate == "iscooked" and literal.variables[0] == food_image_name:
                 food_image_name = "cooked" + food_image_name
-        
+            if literal.predicate == "isfried":
+                if literal.variables[0] == food_image_name:
+                    food_image_name = "fried" + food_image_name
+                elif literal.variables[0] == food_image_name[3:]:
+                    food_image_name = "fried" + food_image_name[3:]
```

Comment:
Add a TODO here that this may need to change if we want to actually have fried potatoes and not french fries.